utopian-io/v2.utopian.io
# Add the Pro Review field to the article form

Fix #155 

### Verify that
- [ ] new functions are prefixed with JSDoc information -> nothing new
- [X] the pull request contains unit tests for new code that passes and has at least 50% coverage -> test modified

---

Please provide a test plan so that the reviewers can see how you tested your work and/or the different acceptance criteria described in the task.

### **Test plan**

_Create article_

* Set the value to false
* Fill the rest

**Expected result:** You should see the proReview set at **false** in the database


_Update the article_

* Set the value to true

**Expected result:** You should see the proReview set at **true** in the database
